### DISCUSSION TITLE
GPU support to the VM in codespace!

### DISCUSSION THREAD

**Discussion by SaladDay** [community/community#51310] [Programming Help] [answered]
### Select Topic Area

Product Feedback

### Body

Am learning machine learning/deep learning and I would like CodeSpace to include GPU support. Otherwise there is no big difference with me knocking code directly with mac

**Comment by sethjuarez** [comment, marked as answer]
Hi there! Seth here. Looks like [GPUs are available](https://github.blog/2022-11-10-whats-new-with-codespaces-from-github-universe-2022/#data-science-ai-and-machine-learning) in limited beta. There's some [documentation](https://docs.github.com/en/codespaces/developing-in-codespaces/getting-started-with-github-codespaces-for-machine-learning) and a [sign-up form](https://github.surveymonkey.com/r/Y75GX9T) if you're interested. Hope this helps!

**Reply by samskiter** [reply]
Hey, survey link not working - are these still available?

**Comment by SaladDay** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
